prov/efa/src/rxr/rxr_pkt_type_data.c

```diff
@@ -231,10 +231,19 @@ void rxr_pkt_handle_data_send_completion(struct rxr_ep *ep,
 	struct rxr_tx_entry *tx_entry;
 
 	tx_entry = (struct rxr_tx_entry *)pkt_entry->x_entry;
-	tx_entry->bytes_acked +=
-		rxr_get_data_pkt(pkt_entry->pkt)->hdr.seg_size;
+	tx_entry->bytes_acked += rxr_get_data_pkt(pkt_entry->pkt)->hdr.seg_size;
 
 	if (tx_entry->total_len == tx_entry->bytes_acked)
+	    if ((!(tx_entry->rxr_flags & RXR_DELIVERY_COMPLETE_REQUESTED)) &&
+		tx_entry->rxr_flags & RXR_RECEIPT_HANDLED)
```

Comment:
Should the condidton be "&&" or "||" ? 

Also place add {} for if conditions.

Reply:
`rxr_pkt_handle_data_send_completion()` has been reimplemented